plugins/logging/logging.cpp

```diff
@@ -75,6 +75,18 @@ LOGGING_API void LOGGING_CALL logDbgLog(unsigned srcLen, const char * src)
     }
 }
 
+LOGGING_API void LOGGING_CALL logDbgLogV2(ICodeContext *ctx, unsigned srcLen, const char * src)
+{
+    StringBuffer log(srcLen, src);
+    StringArray loglines;
+    log.replace('\r', ' ');
+    loglines.appendList(log, "\n", false);
```

Comment:
Slightly debating whether this should trim spaces or not (in combination with the line above).  I don't think it matters though.